content/blog/2024/06/welcome-cel-policy.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Introducing the CEL policy
+authors:
+  - Victor Cuadrado Juan
+date: 2024-06-14
+---
+
+We are pleased to announce a new policy by the Kubewarden team: **cel-policy**.
+
+This new policy uses the [Common Expression Language (CEL)](https://cel.dev).
+For those new to CEL, it is a general-purpose expression language designed to
+be fast, portable, and safe to execute. CEL as a language is memory-safe,
+side-effect free, terminating (as in "programs cannot loop forever"), and strong &
+dynamically typed.
+
+CEL is a perfect candidate for extending the Kubernetes API, as CEL expressions
+can be easily inlined into CRD schemas, and compiled and type-checked
+"ahead-of-time" (when CRDs are created and updated). With Kubernetes 1.30, CEL features such as
+[ValidatingAdmissionPolicies](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy)
+and other validation rules are now marked as stable.
+
+## cel-policy: What's inside
+
+For including CEL functionality, Kubernetes API server uses the upstream
+[cel-go](https://pkg.go.dev/github.com/google/cel-go). In addition, Kubernetes
+also provides different CEL libraries that live in `apiextensions-apiserver`
+and enrich the CEL experience. These range from standard functions and macros,
+to supplemental functions for strings, lists, regex, URls, etcetera.
+
+In Kubewarden, policies are Wasm modules. And in particular, we have [WASI
+policies](https://github.com/kubewarden/docs/pull/414). Hence with all the
+pieces falling in place, it is only logical that we compile both the usptream CEL Go interpreter
+Go and the additional Kubernetes CEL libraries, and ship it as a Go WASI
+policy: cel-policy. You can find it as always in
+[artifacthub.io](https://artifacthub.io/packages/kubewarden/cel-policy/cel-policy):
```

Comment:
check, and embed artifacthub widget when available.

Reply:
decided against it, it doesn't look good in my opinion:
![image](https://github.com/kubewarden/kubewarden.io/assets/2196685/47bd206b-3a1c-42cb-9b3d-00b52d27d948)